frontend/src/containers/EditChart.tsx

```diff
@@ -619,19 +614,19 @@ function EditChart() {
       ) : (
         <div className="grid-row">
           <div className="grid-col-12">
-            <form
-              onSubmit={handleSubmit(onSubmit)}
-              aria-labelledby="editChartFormHeader"
+            <div
+              id="panel1"
+              hidden={step !== 0}
+              role="tabpanel"
+              tabIndex={0}
+              aria-labelledby="editChartFormHeaderStep1_chooseDdata"
             >
-              <div
-                id="panel1"
-                hidden={step !== 0}
-                role="tabpanel"
-                tabIndex={0}
-                aria-labelledby="editChartFormHeaderStep1"
+              <form
```

Comment:
big kudos on splitting the form each per tab. I had that on my personal backlog lol

Reply:
nit: now, that you moved the submit to each individual form I think we should create a header on top of the form and have the header render once and each form within a panel